schema/deploy/shipping/views.sql

```diff
@@ -38,8 +38,10 @@ create view shipping.incidence_model_observation_v1 as
              else (encounter.details->'age'->>'value')::int
            end as age,
 
-           age_bin_fine.range as age_range_fine,
-           age_bin_coarse.range as age_range_coarse,
+           lower(age_bin_fine.range) as age_range_fine_lower,
+           upper(age_bin_fine.range) as age_range_fine_upper,
+           lower(age_bin_coarse.range) as age_range_coarse_lower,
+           upper(age_bin_coarse.range) as age_range_coarse_upper,
```

Comment:
Let's leave the existing range fields too? This makes it clear if the lower/upper bounds are inclusive or exclusive.